cdk/bin/cdk_state.ts

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import aws = require("aws-sdk");
+import fs = require("fs");
+import toml = require("@iarna/toml");
+
+const cf = new aws.CloudFormation()
+
+!(async () => {
+  const { Exports = [] } = await cf.listExports().promise()
```

Comment:
I did both outputs and exports for now. I don't know that I have a deep enough understanding to make an idiomatic recommendation on the differences. Whichever you think is better.

The thing I would make sure of is that they're in a predictable table, like `cfn_outputs` or `cfn_exports` or similar, so that they can be accessed as (for example) `documentTableName = cfn_outputs.${document_bucket.document_table.output}`